core/modules/user/user.module

```diff
@@ -1350,23 +1350,20 @@ function user_menu() {
     'page callback' => 'backdrop_get_form',
     'page arguments' => array('user_login'),
     'access callback' => 'user_is_anonymous',
-    'type' => MENU_DEFAULT_LOCAL_TASK,
```

Comment:
If you remove the type, the default is MENU_NORMAL_ITEM, which means that it will automatically show up.

Maybe MENU_CALLBACK, possibly MENU_SUGGESTED_ITEM is the better choice here.